.ruby-version

```diff
@@ -0,0 +1 @@
+jruby-9.1.13.0
```

Comment:
there's no need to add this file, in the [.travis.yml](https://github.com/logstash-plugins/logstash-input-exec/blob/master/.travis.yml#L7) we already set rvm to use jruby